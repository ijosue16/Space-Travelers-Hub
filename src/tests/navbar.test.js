import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/configurestore';
import NavBar from '../components/Navbar/navbar';

it('Navbar renders correctlly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <BrowserRouter>
          <NavBar />
        </BrowserRouter>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
