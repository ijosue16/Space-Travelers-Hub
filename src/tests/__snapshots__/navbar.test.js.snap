// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing navbar Navbar renders correctlly 1`] = `
<nav
  className="navbar bg-body-tertiary d-flex flex-row mx-3 py-1 border-2 border-bottom mb-5  "
>
  <div
    className="navbar-brand d-flex align-items-center gap-2"
  >
    <img
      alt="Logo"
      className="d-inline-block align-text-top"
      height="60"
      src="planet.png"
      width="60"
    />
    <h2
      className="m-0 text-wrap"
    >
      Space traveler's Hub
    </h2>
  </div>
  <ul
    className="navbar-nav d-flex flex-row gap-3 text-end align-items-center nav-pills "
  >
    <a
      aria-current="page"
      className="rockets"
      href="/"
      onClick={[Function]}
    >
      Rockets
    </a>
    <a
      className="text-decoration-none"
      href="/Missions"
      onClick={[Function]}
    >
      Missions
    </a>
    <div>
      |
    </div>
    <a
      className="text-decoration-none"
      href="/Myprofile"
      onClick={[Function]}
    >
      My profile
    </a>
  </ul>
</nav>
`;
